.github/workflows/prune.yml

```diff
@@ -15,14 +16,37 @@ jobs:
     runs-on: ubuntu-latest
 
     steps:
-      - name: Prune
-        uses: snok/container-retention-policy@v2
+      # Required in order to access script files in this repository
+      - uses: actions/checkout@v4
+
+      # Prevents multi-platform release images from being pruned by identifying all manifest lists
+      - name: Fetch multi-platform package version SHAs
```

Comment:
A: For consistency with the snapshot step, it might make sense to call this _Fetch release version SHAs_ and set the id/output names to be `release-digests`